docs/src/howToGetStarted.md

```diff
@@ -38,105 +281,267 @@ for f in dataFiles
 end
 ```
 
-Read meta-data and panel as a `DataFrame`, and make sure that the column names match the CyTOF
-FCS file names:
+The metadata is present in external files; we read it into a DataFrame and
+extract information about FCS data columns from there:
 
 ```julia
-# Read  files as DataFrames
+using GigaSOM
+
+# Read the dataset description in XLSX files as DataFrames
 md = GigaSOM.DataFrame(GigaSOM.XLSX.readtable("PBMC8_metadata.xlsx", "Sheet1")...)
 panel = GigaSOM.DataFrame(GigaSOM.XLSX.readtable("PBMC8_panel.xlsx", "Sheet1")...)
-panel[:Isotope] = map(string, panel[:Isotope])
-panel[:Metal] = map(string, panel[:Metal])
-panel[:Antigen] = map(string, panel[:Antigen])
-panel.Metal[1]=""
-GigaSOM.insertcols!(panel,4,:fcs_colname => map((x,y,z)->x.*"(".*y.*z.*")".*"Dd",panel[:Antigen],panel[:Metal],panel[:Isotope]))
-print(panel.fcs_colname)
+
+# Get the parameters structure from the first FCS files
+_, fcsParams = loadFCSHeader(md[1, :file_name])
+
+# See what antigens are saved in there
+_, fcsAntigens = getMarkerNames(getMetaData(fcsParams))
+
+# See which antigens we want to use (assume we want only the lineage markers
+antigens = panel[panel[:,:Lineage].==1, :Antigen]
+
+# Make the names a bit more Julia-friendly and predictable:
+cleanNames!(antigens)
```

Comment:
Maybe all these code lines and comments should be separate rows in the notebook.